1_The_Physical/Getting_Started_Emergent_Design/1_Red_Green_Refactor/1_1_Fizzbuzz/src/fizzbuzz.ts

```diff
@@ -0,0 +1,3 @@
+export function fizzBuzz (number:number) {
```

Comment:
One minor suggestion for improvement — to avoid using a keyword from the programming language as a variable name, perhaps you could consider naming the input input or num instead of number.